packages/backend/src/models/MembershipPlan.ts

```diff
@@ -14,11 +14,11 @@ import { Club } from './Club';
 import { Organisation } from './Organisation';
 
 export enum PlanType {
-  weekly,
-  monthly,
-  quarterly,
-  biannual,
-  annual,
+  hebdomadaire,
+  mensuel,
+  trimestriel,
+  semestriel,
+  annuel,
 }
```

Comment:
No please stop putting french in the code.... This is an enumerator so it will be number at end. We can always make a layer that translate it after !